react/ConditionLayoutBinding.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
```

Comment:
Hey @cristian-milea, we prefer to use interfaces instead of types. Can you change all these types to interfaces?

Reply:
I just use the types from the original condition-layout. Changing them is not in the scope of this PR, as it includes changes to the condition-layout for product. This should be a different PR